EveripediaNetwork/ep-ui
# Mobile Nav Menu Navigation

# Mobile Menu Navigation Fixes

_This PR fixes the layout of the mobile menu and properly aligns the elements of the mobile nav menu, fixes nav layout on the wallet drawer and ensures mobile nav menu shows on all necessary breakpoints_

## How should this be tested?

1. Compare old nav menu layout to the new one and ensure every element is properly aligned.
2. Check to ensure mobile nav menu now shows on tab breakpoints 

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3122)github.com/EveripediaNetwork/issues/issues/
